src/commands/set.json

```
@@ -23,6 +23,10 @@
             [
                 "7.0.0",
                 "Allowed the `NX` and `GET` options to be used together."
+            ],
+            [
+                "8.2.0",
```

Our next version is `8.1.0`.

```suggestion
                "8.1.0",
```